### PR TITLE
docs for warehouse cost

### LINKED ISSUE #1261: [DOCUMENTATION] Docs for DBSQL costs

### PR DESCRIPTION
**Pull request by aman-db**
closes #1261

### CODE REVIEW

**Review by neilbest-db — CHANGES_REQUESTED** [review]
Please see comments on respective files.

File: docs-compose/overwatch_docs/content/DataEngineer/Definitions/_index.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Complete column descriptions are only provided for the consumption layer. The en
 * [sparkStream](#sparkstream)
 * [warehouse](#warehouse)
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DataEngineer/Definitions/_index.md:69]: done

File: docs-compose/overwatch_docs/content/DataEngineer/Definitions/_index.md [hunk 2]
```diff
@@ -942,32 +944,119 @@ to improve performance.
...
+| custom_tags          | string | JSON string of key/value pairs for all cluster associated custom tags give to the cluster           |
+| node_type_id         | string | Worker Node type for the compute asset (not supported for Warehouses yet)                           |
+| node_count           | long   | Cluster worker node count                                                                           |
+| response             | struct | HTTP response including errorMessage, result, and statusCode                                        |
+| user_agent           | string | Request origin such as browser, terraform, api, etc                                                 |
+| unixTimeMS           | long   | Unix time epoch as a long in milliseconds                                                           |
+| Pipeline_SnapTS      | string | Snapshot timestamp of Overwatch run that added the record                                           |
+| Overwatch_RunID      | string | Overwatch canonical ID that resulted in the record load                                             |
+
+#### warehouseDbuDetails
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DataEngineer/Definitions/_index.md]: done

File: docs-compose/overwatch_docs/content/DataEngineer/Definitions/_index.md [hunk 3]
```diff
@@ -942,32 +944,119 @@ to improve performance.
...
+| total_dbu    | long    | Total dbus charged for the cluster per hour                                                                                                                                                                                                                         | 
+| is_active    | boolean | whether the contract price is currently active. This must be true for each key where activeUntil is null                                                                                                                                                            |
+| activeFrom*  | date    | The start date for the costs in this record. **NOTE** this MUST be equal to one other record's activeUntil unless this is the first record for these costs. There may be no overlap in time or gaps in time.                                                        |
+| activeUntil* | date    | The end date for the costs in this record. Must be null to indicate the active record. Only one record can be active at all times. The key (API_name) must have zero gaps and zero overlaps from the Overwatch primordial date until now indicated by null (active) |
+
+#### WarehouseStateFact
```
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/content/DataEngineer/Definitions/_index.md:960]: done

File: docs-compose/overwatch_docs/static/images/_index/Overwatch_Gold_0.8.2.0.png
[the remark this answers is not in the document]
Comment (aman-db) [reply docs-compose/overwatch_docs/static/images/_index/Overwatch_Gold_0.8.2.0.png:1]: let me know if it needs to be removed we need to make changes in the lucid chart

File: docs/images/_index/Overwatch_Gold_0.8.2.0.png
[the remark this answers is not in the document]
Comment (aman-db) [reply docs/images/_index/Overwatch_Gold_0.8.2.0.png:1]: no , we need to make changes in the lucid chart , export it as a image and use here

**Review by neilbest-db — APPROVED** [review]

**Pull request merged**